### PR TITLE
Env

### PR DESCRIPTION
**Pull request by mkorszun**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -158,6 +161,7 @@ Options:
 	bucketName := arguments["<bucket>"].(string)
 	fileNames := arguments["<name>"].([]string)
 	envs := arguments["--envs"]
+	env := arguments["--env"].(string)
```
Comment (fern4lvarez) [inline main.go]: For more readability and less ambiguity, I'd rename `env` var to `envName`.

**Comment by mkorszun** [comment]
@fern4lvarez done.

File: README.md [hunk 2]
```diff
@@ -41,12 +41,19 @@ Create a `~/.gs3pload/envs.json` file with desired environments:
...
 
-If you want to use a custom environments file, you can use the `--config` flag:
+If you want to use a custom environments file, you can use the `--envs` flag:
 
 ~~~
-$ gs3pload push --custom /path/to/mycustomenvs.json bucket file
+$ gs3pload push --envs /path/to/mycustomenvs.json bucket file
 ~~~
 
+If for some reason yout want to push only to single environment, use `--env` flag:
```
Comment (fern4lvarez) [inline README.md]: you want

**Comment by fern4lvarez** [comment]
Thanks for your PR! :tada:

**Pull request merged**